### DISCUSSION TITLE
Studies inferring infection dynamics from deaths with estimate_infections()?

### DISCUSSION THREAD

**Discussion by adamkucharski** [epiforecasts/EpiNow2#529] [General]
Was recently having another look through [this paper](https://academic.oup.com/biometrics/article/78/3/1127/7477180) using a GAM approach for infection estimation and wondered if there’s been any research papers using the estimate_infections() from EpiNow2 to infer infection dynamics from deaths? Know there was a lot of work on cases, but anything with more lagged (but perhaps less reporting-dependent) outcomes?

Had a go at coding up a rough simulation recovery example in the process, partly to see what performance looked like, and partly to see where if there were any obvious places for integration/streamlining with other tools might be (e.g. epiparameter, incidence2). [Gist is here](https://gist.github.com/adamkucharski/11227d7ccca4b8e88e3aa9d1f4cd6963) with 'OBSERVATIONS' highlighting some thoughts on links with other tools (edited from earlier version to use `epinow` and fix distribution assumptions - adapting some implementation by @avallecam [here](https://github.com/epiverse-trace/howto/issues/34).

![delay_recover](https://github.com/epiforecasts/EpiNow2/assets/8329046/eb20f96b-eb21-41e4-bd19-37d2b5e6b123)

**Comment by sbfnk** [comment]
Thanks! I'm not aware of any papers that did that but it's a nice use case that might be good for a vignette / doc - one thing to note is that for this use case (where you're not interested in real time estimates) you could use the [nonmechanistic infection model](https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections.html#non-mechanistic-infection-model) by setting `rt = NULL` in the call to `epinow()` which should speed things up substantially whilst giving comparable results.

**Comment by seabbs** [comment]
Thanks @adamkucharski. The persistent underestimation here makes me think there is a mismatch between the simulation and the `EpiNow2` model as its not clear that this would be likely as bias from the model otherwise. Looking at the code though its not totally clear to me what could be driving this (internal sims at CDC on a SEIR with long death delays and a piecewise constant Rt end up with pretty on-point recovery hence the surprise at above).

Agree with Seb's suggestion to make use of the deconvolution approach if this is the interest. That will still potentially slightly struggle here as it assumes a log link whereas the simulated data looks like its linear/constant but it should do better than assuming a renewal generative process.

**Comment by adamkucharski** [comment]
Agree a linear function may be too much of a pathological example given it doesn't have a plausible underlying epidemic process. Re-ran with exponential growth/decline (using the nonmechanistic infection model Seb added to that gist), albeit still with a very peaky dynamic that is probably just tricky for deconvolution approaches in general....
![delay_recover](https://github.com/epiforecasts/EpiNow2/assets/8329046/dd4a4876-521e-40cc-9e9f-1cdc2a2742e5)

Also ran on the UK COVID dashboard deaths out of curiosity (you may have done this previously, can't remember all the early iterations of R work!) Estimates for fatal infections shown with 16th and 23rd March overlaid. Interestingly (but perhaps not surprisingly) it's a bit of a bumpier inferred curve than the GAM based method in the Wood paper, but with a similar peak timing.

![delay_recover_uk](https://github.com/epiforecasts/EpiNow2/assets/8329046/eca94ea8-3acb-4b9c-8a5b-d61ca844540e)

**Comment by adamkucharski** [comment]
Follow up observation - obviously the above infection curve is sensitive to the assumption about onset-to-death, which is commonly cited as 17-20 days (e.g. median in [Linton et al,](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7074197/) adjusting for truncation, is 17 days). But in [this Co-CIN report](https://www.gov.uk/government/publications/co-cin-covid-19-time-from-symptom-onset-until-death-in-uk-hospitalised-patients-7-october-2020) on data from first wave in UK implies median 13 days (analysed in autumn 2020, so shouldn't be subject to truncation?). Can't find any more comprehensive studies reporting distributions from UK. If we shift delay a few days shorter, we shift the conclusion of peak timing – seems like a crucial estimate about UK dynamics to pin down...

**Reply by seabbs** [reply]
The linton et al. estimate is probably a bit unreliable as we have been unable to justify the method used for right truncation adjustment (its likely to induce bias) - more on that in the recent preprint on delay estimation. 

> [this Co-CIN repor](https://www.gov.uk/government/publications/co-cin-covid-19-time-from-symptom-onset-until-death-in-uk-hospitalised-patients-7-october-2020)

I can't tell from this how sampling was structured but your right its likely not impacted by truncation (depending on how they determined 1st wave).
